src/main/java/com/powsybl/openloadflow/ac/equations/AbstractBranchAcFlowEquationTerm.java

```diff
@@ -34,9 +34,6 @@
     protected AbstractBranchAcFlowEquationTerm(LfBranch branch) {
         this.branch = Objects.requireNonNull(branch);
         PiModel piModel = branch.getPiModel();
-        if (piModel.getR() == 0 && piModel.getX() == 0) {
```

Comment:
I falled in that exception during my tests so I removed it.

Reply:
Maybe we can just for here to replace by a min low impedance line as it could be complex and write an issue about it.